.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -2,17 +2,20 @@
 name: Specified Feature
 about: Agreed specification for a feature or enhancement
 title: ''
-labels: 'feature'
+labels: 'feature request'
 assignees: ''
 ---
 
-<!-- Thanks for specifying a new feature! -->
+**Provide context**
+<!-- What's the status quo and how is this non-optimal?** -->
```

Comment:
I don't think this is clear enough (at least for me "giving context" could mean anything). Can you formulate the headline as question too?

Maybe: What is the current behaviour and why should it be changed?
Comment: Please give a precise description of the status quo and how this is not optimal.

Reply:
Updated (more or less verbatim).